fuzz/type_info_adt_set.h

```diff
@@ -21,10 +21,10 @@ struct set_op {
 	enum set_op_type t;
 	union {
 		struct {
-			uintptr_t item;
+			uintptr_t item; /* 0-based */
```

Comment:
Why did you decide to keep the instance 0-based and shift it everywhere it's used? Either approach makes sense, though I probably would have shifted it in the generator instead -- that way I wouldn't have to worry about forgetting to offset it anywhere.

Reply:
Because I'm mentally distinguishing the abstract element number (as indexed into the model's bitmap) from the corresponding pointer value. My reasoning is that the only dirty part here is in casting one to the other - so the `(void *)` is the part where I think workarounds belong.

As a thought experiment, I modelled that as indexing into some array of valid non-NULL pointers: `a[item]`. There the rest of the code would be unchanged, and the offset would be "in" each of the pointer values.

`THEFT_TRIAL_SKIP` is a great idea - I'll try that.